WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/InPersonPaymentsLearnMore.swift

```diff
@@ -4,19 +4,21 @@ struct InPersonPaymentsLearnMore: View {
     @Environment(\.customOpenURL) var customOpenURL
 
     private let viewModel: LearnMoreViewModel
+    private let showInfoIcon: Bool
 
-    init(viewModel: LearnMoreViewModel = LearnMoreViewModel()) {
+    init(viewModel: LearnMoreViewModel = LearnMoreViewModel(),
+         showInfoIcon: Bool = true) {
```

Comment:
Minor thing here, perhaps that showInfoIcon should be moved to the view model?

Reply:
Yep, sounds good, I'll do that in a future PR